_posts/2024-10-23-arrow-go-18.0.0-release.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "Apache Arrow Go 18.0.0 Release"
+date: "2024-10-23 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the v18.0.0 release of
+Apache Arrow Go. This release covers 35 resolved issues from
+11 contributors.
+
+## Release Highlights
+
+### Move to new Repository
+
+The Apache Arrow Go implementation has been shifted out of the Arrow monorepo and to its
+own repository now located at https://github.com/apache/arrow-go. This will enable fewer
+major version releases and facilitate more minor version/patch releases instead as it will
+detach the version number from the Arrow C++ and other implementations in the monorepo.
+
+Existing Go related issues have been moved to the new repository and please file any new
+issues on the new repository instead of the Arrow monorepo.
+
+### Go Version
+
+With this release, the minimum Go version is now 1.22
+
+### Arrow
+
+#### Canonical Extension Types
+
+There is a new [`extensions`](https://pkg.go.dev/github.com/apache/arrow-go/v18@v18.0.0/arrow/extensions) package 
+which implements [canonical extension types](https://arrow.apache.org/docs/format/CanonicalExtensions.html). This
+includes:
+
+* Opaque [GH-43455](https://github.com/apache/arrow/issues/43455)
+* JSON and UUID [GH-43764](https://github.com/apache/arrow/issues/43764)
+* Bool8 [GH-17682](https://github.com/apache/arrow/issues/17682)
+
+#### Features
+
+* Added the initial implementation for the Decimal32/Decimal64 data types [Go-120](https://github.com/apache/arrow-go/issues/120)
+* Added `is_null`, `is_not_null` and `is_nan` function kernels to the `compute` package [GH-134](https://github.com/apache/arrow-go/issues/134)
```

Comment:
nit: inconsistency over using `Go-NNN` vs `GH-NNN` vs `GO-NNN` for `arrow-go` issues

Reply:
Both issues here are on arrow-go, though, is what I'm saying. (Go-120 vs GH-134)

Reply:
accepted Kou's sugestions which alleviated the discrepency